packages/amplication-plugin-api/src/pluginVersion/npm-plugin-version.service.ts

```diff
@@ -3,13 +3,35 @@ import { Plugin, PluginVersion } from "../../prisma/generated-prisma-client";
 import { AmplicationLogger } from "@amplication/util/nestjs/logging";
 import { Packument } from "pacote";
 import { NpmService } from "../npm/npm.service";
+import { ConfigService } from "@nestjs/config";
 
 @Injectable()
 export class NpmPluginVersionService {
+  private readonly versionFilterRegex: RegExp;
+
   constructor(
     @Inject(AmplicationLogger) readonly logger: AmplicationLogger,
-    private readonly npmService: NpmService
-  ) {}
+    private readonly npmService: NpmService,
+    private readonly configService: ConfigService
+  ) {
+    const pluginStableVersionRegex =
```

Comment:
Can you add a comment to describe an example of what each regex will match?

Reply:
I expected a regex to match (*).beta(*)

Reply:
it can be anything as long as it is a valid semver version, not specifically "beta"